apps/studio/prisma/schema.prisma

```diff
@@ -46,16 +56,25 @@ model Resource {
 
   permission Permission[]
 
-  mainBlob   Blob?   @relation("MainBlob", fields: [mainBlobId], references: [id])
-  mainBlobId BigInt? @unique
+  versionId BigInt?
```

Comment:
question: should this also be `@unique` too?

Reply:
Each resource to a unique version makes sense, will add!